Make Root-VSIX work on machines with different cultures

MortenHoustonLudvigsen opened the pull request:
Make Root-VSIX work on machines with different cultures

SLaks commented:
What exactly is the problem?
The correct way to fix this is to pass invariant culture only where needed. (eg, `TryParse`)

MortenHoustonLudvigsen commented:
@SLaks You are absolutely correct. I was a bit hasty.

The problem was that in Denmark, where I live, we use a comma as the decimal point, and a full stop as the thousands separator. The following line therefore fails to parse version numbers correctly:

````csharp
if (!decimal.TryParse(s, out v))
````

I have changed this line to use the invariant culture and removed the code that I added in the first commit.

By the way, thank you so much for this tool. It has made my work on the KarmaTestAdapter much easier.

Regards, Morten
